packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -24,6 +24,7 @@ import { PublisherBase, PublishRequest, TechDocsMetadata } from './types';
 import fs from 'fs-extra';
 import { Readable } from 'stream';
 import JSON5 from 'json5';
+import limiterFactory from 'p-limit';
```

Comment:
Bit ridiculous nit tbh, but could call this `createLimiter` 😅

Reply:
Done!